main.go

```diff
@@ -1,54 +1,45 @@
 package main
 
 import (
-	"fmt"
+	"os"
+	"os/signal"
 	"rest-gateway/conf"
+	"rest-gateway/handlers"
 	"rest-gateway/logger"
 	"rest-gateway/router"
+	"syscall"
 	"time"
-
-	"github.com/memphisdev/memphis.go"
 )
 
-func main() {
+func initalizeLogger() {
 	configuration := conf.GetConfig()
-	var conn *memphis.Conn
 	ticker := time.NewTicker(1 * time.Second)
 	for {
 		select {
 		case <-ticker.C:
-			var err error
-			opts := []memphis.Option{memphis.Reconnect(true), memphis.MaxReconnect(10), memphis.ReconnectInterval(3 * time.Second)}
+			creds := configuration.CONNECTION_TOKEN
```

Comment:
the only thing you should do inside the for loop is the connection not all the rest